osu.Game/Overlays/Profile/Header/Components/ProfileRulesetTabItem.cs

```diff
@@ -23,7 +23,7 @@ public bool IsDefault
 
                 isDefault = value;
 
-                icon.FadeTo(isDefault ? 1 : 0, 200, Easing.OutQuint);
+                icon.Alpha = isDefault ? 1 : 0;
```

Comment:
Any particular reason why this transition was removed?

Reply:
Because `AlwaysPresent = true` has been removed as well to match design, means that the item with the star visible will only have its width extended rather than all items. Keeping the fade transition will cause it to look jarring.

Reply:
To show the jarringness:

with transition:

https://user-images.githubusercontent.com/22781491/166100340-6b506838-6526-4727-b403-9c23ea08ab02.mp4 

no transition:

https://user-images.githubusercontent.com/22781491/166100008-71b3c49f-c2b3-4d7d-b356-7c544a0fb416.mp4